### PR TITLE
Add a file for Lino

### PR DESCRIPTION
**Pull request by cheri-tolkinen**
The purpose of this pull request if to add a hometown file for Lino Lakes MN

Please answer these questions:
Have you been to Lino Lakes?
Do you like to watch hockey games?
What kind of food do you like?

### CODE REVIEW

**Comment by MichaelSJohnson** [comment]
I have been to Lino and like hockey games.
My favorite food is Italian

File: cheri-tolkinen-lino.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Lino Lakes MN
+
+### Places the locals like
+#### Centennial Arena
+
+### Restaurants
```
Comment (MichaelSJohnson) [inline cheri-tolkinen-lino.md:6]: Are there any good Italian restaurants

Comment (cheri-tolkinen) [reply cheri-tolkinen-lino.md:6]: No.  You'll have to go to Roseville for Italian.

**Comment by cheri-tolkinen** [comment]
This pull request resolves #14

**Review by MichaelSJohnson — APPROVED** [review]
changes look good :smiley:

**Pull request merged**